server/chat-plugins/helptickets.ts

```diff
@@ -2159,6 +2265,19 @@ export const commands: Chat.ChatCommands = {
 		},
 		deletehelp: [`/helpticket delete [user] - Deletes a user's ticket. Requires: &`],
 
+		logs(target, room, user) {
+			this.checkCan('lock');
+			const [targetString, dateString] = Utils.splitFirst(target, ',').map(i => i.trim());
+			const id = toID(targetString);
+			if (!id) return this.errorReply(`Specify a userid.`);
+			return this.parse(`/j view-help-logs-${id}${dateString ? `--${dateString}` : ''}`);
+		},
+		logshelp: [
+			`/helpticket logs [userid][, date] - View logs of the [userid]'s text tickets. `,
+			`If a [date] is given, searches only that month.`,
```

Comment:
```suggestion
			`/helpticket logs [userid][, month] - View logs of the [userid]'s text tickets. `,
			`If a [month] is given, searches only that month.`,
```

Reply:
[17:48] mia: you can commit that review to my PR btw
[17:49] mia: I'm on phone so it's hard